src/core/Clock.js

```diff
@@ -1,3 +1,9 @@
+try {
+  var NODE_JS = typeof process.versions.node === 'string';
```

Comment:
Browsers don't have file scope, so this creates a global variable when run in a browser. It also throws and catches an exception. We'd like to avoid doing that as much as possible because it makes it harder for users to debug their own code using exception breakpoints. Something like `typeof process !== 'undefined' && process.versions && typeof process.versions.node === 'string'` would work better than try/catch. (Here and in ClockSpec.js as well.)